pom.xml

```diff
@@ -203,16 +203,27 @@
                         <!-- io.netty:netty-handler: https://ossindex.sonatype.org/vulnerability/sonatype-2020-0026
                         Not a vulnerability in netty but how it is configured by the client com.azure:azure-core-http-netty. -->
                         <exclude>sonatype-2020-0026</exclude>
-                        <!-- io.netty:netty-common: https://ossindex.sonatype.org/vulnerability/CVE-2022-24823
-                        Excluded because we assume that the attacker does not have local access to the machine.
-                        Can be removed once com.azure:azure-core-http-netty upgrades to a newer netty version. -->
-                        <exclude>CVE-2022-24823</exclude>
-                        <exclude>CVE-2022-2047</exclude>
-                        <exclude>CVE-2022-30187</exclude>
                         <!-- org.yaml.snakeyaml (1.33) https://ossindex.sonatype.org/vulnerability/CVE-2022-38750
                         Currently not fixed in latest version, however a test dependency and not used on untrusted files.
                         Can be removed once org.yaml.snakeyaml upgrades to a newer version. -->
                         <exclude>CVE-2022-38752</exclude>
+                        <!-- org.apache.hadoop:hadoop-common:jar:3.3.4 -->
+                        <!-- Rationale for exclusion: No update available, yet. -->
+                        <exclude>sonatype-2022-5820</exclude>
+                        <!-- org.apache.hadoop:hadoop-hdfs-client:jar:3.3.4 -->
+                        <!-- Rationale for exclusion: No update available, yet. -->
+                        <exclude>sonatype-2022-5732</exclude>
+                        <!-- com.fasterxml.jackson.core:jackson-databind:jar:2.13.4 -->
+                        <!-- Rationale for exclusion: No update available, yet. -->
+                        <exclude>CVE-2022-42003</exclude>
```

Comment:
CVE-2022-42003 can probably be fixed by adding a direct dependency to `com.fasterxml.jackson.core:jackson-databind 2.13.4.2`: 
```xml 
<dependency>
    <groupId>com.fasterxml.jackson.core</groupId>
    <artifactId>jackson-databind</artifactId>
    <version>2.13.4.2</version>
</dependency>
```

Reply:
Good eye. Done with success.